templates/blog/newsletter-form.html

```diff
@@ -1,96 +1,86 @@
 <div class="p-card--muted">
-    <header class="p-card__header">
-      <h5 class="p-muted-heading">Newsletter signup</h5>
-    </header>
+  <header class="p-card__header">
+    <h5 class="p-muted-heading">Newsletter signup</h5>
```

Comment:
Again, I think this is just how it is on the u.c press centre & blog, but the heading heirarchy is all over the map on the blog post view:
![Screenshot from 2022-04-01 11-01-16](https://user-images.githubusercontent.com/2376968/161241976-6988fd76-857f-4183-8f6f-8dff05fc654b.png)

It should be:
- 1: Title of blog post
  - 2: Newsletter signup
    - 3: Select topics you're interested in 
  - 2: Related blog posts
    - 3: Title of related post
    - 3: Title of related post
    - 3: Title of related post

And I don't think the author "Canonical" should be a heading element at all, that could be a `p` element, with a `.p-heading--3` if necessary.